### PR TITLE
docs(table-libraries): added basic info about tegel, tanstack and ag-grid

### PR DESCRIPTION
**Pull request by theJohnnyMe**
## **Describe pull-request**  
Basic info on differences and when to use each library for table

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3449](https://tegel.atlassian.net/browse/CDEP-3449)    

## **How to test** 
1. Check text written under Intro --> Announcements --> Table Libraries
2. Please suggest changes if something in not current or wrong with grammar or spell check.

[CDEP-3449]: https://tegel.atlassian.net/browse/CDEP-3449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/core/src/stories/announcements/table-libraries.stories.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                    <h5>Tegel demo page examples:</h5>
+                    <ul>
+                        <li><a href="https://github.com/scania-digital-design-system/tegel-angular-17-demo">Tegel Angular Demo</a></li>
+                        <li><a href="https://github.com/scania-digital-design-system/tegel-react-demo">Tegel React Demo</a></li>
+                    </ul>
+                    </section>
+                    <section>
+                    <h2 class="tds-headline-02">Tegel & Tanstack</h2>
+                    <p> Tanstack is a headless library for building powerful tables in various frameworks or plain JavaScript. On our demo pages, we have provided examples of how to combine the Tegel Table component with Tanstack. Check out the column filtering and data editing features. </p>
+                    <p> Tanstack is <b>free to use</b> and is available on npm. Its documentation, supported frameworks, and fatures examples can be found here: <a href="https://tanstack.com/table/latest">Tanstack Table Documentation</a>. </p>
```
Comment (nathalielindqvist) [inline packages/core/src/stories/announcements/table-libraries.stories.ts:66]:
```suggestion
                    <p> Tanstack is <b>free to use</b> and is available on npm. Its documentation, supported frameworks, and features examples can be found here: <a href="https://tanstack.com/table/latest">Tanstack Table Documentation</a>. </p>
```

File: packages/core/src/stories/announcements/table-libraries.stories.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    <p> The Tegel Table component is designed to be a flexible and customizable table solution. It does not include built-in logic but instead relies on events and user-defined custom logic. On our demo pages, we have provided examples of how to implement basic logic for the Tegel Table. </p>
+                    <h5>Highlights of Tegel Table:</h5>
+                    <ul>
+                        <li>Part of the Tegel Design System, free to use.</li>
+                        <li>Simple UI components with custom events, props, and slots to accommodate different use cases and tech stacks within Scania.</li>
+                        <li>Sorting, multiselect, filtering, and pagination features with logic are presented on the demo pages.</li>
+                    </ul>
+                    <h5>What do I need to install to use Tegel Table?</h5>
+                    <ul>
+                        <li>Tegel Table is part of the Tegel Design System, so no additional installation is needed.</li>
```
Comment (timrombergjakobsson) [inline packages/core/src/stories/announcements/table-libraries.stories.ts:54]: I'm thinking, to say "Tegel Table" is redundant, as it's name is not "Tegel Table", merely referring to it as the "Table component" suffice.

**Review by nathalielindqvist — APPROVED** [review]
Great work! Should we update the links to the demo pages once the AG-Grid and TanStack examples have been merged?

File: packages/core/src/stories/announcements/table-libraries.stories.ts (hunk 2 above)
Comment (timrombergjakobsson) [inline packages/core/src/stories/announcements/table-libraries.stories.ts:54]:
```suggestion
                        <li>The Table component is part of the Tegel Design System, so no additional installation is needed.</li>
```

File: packages/core/src/stories/announcements/table-libraries.stories.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+            .mb-72 {
+                margin-bottom: 72px;
+            }
+            </style>
+
+            <article class="tds-u-p2 tds-body-01">
+                <section>
+                    <h2 class="tds-headline-02">Tegel Table</h2>
+                    <p> The Tegel Table component is designed to be a flexible and customizable table solution. It does not include built-in logic but instead relies on events and user-defined custom logic. On our demo pages, we have provided examples of how to implement basic logic for the Tegel Table. </p>
```
Comment (timrombergjakobsson) [inline packages/core/src/stories/announcements/table-libraries.stories.ts:45]:
```suggestion
                    <p> The Table component is designed to be a flexible and customizable table solution. It does not include built-in logic but instead relies on events and user-defined custom logic. On our demo pages, we have provided examples of how to implement basic logic for the Tegel Table. </p>
```

File: packages/core/src/stories/announcements/table-libraries.stories.ts [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                        <li><a href="https://github.com/scania-digital-design-system/tegel-react-demo">Tegel React Demo</a></li>
+                    </ul>
+                    </section>
+                    <section>
+                    <h2 class="tds-headline-02">Tegel & Tanstack</h2>
+                    <p> Tanstack is a headless library for building powerful tables in various frameworks or plain JavaScript. On our demo pages, we have provided examples of how to combine the Tegel Table component with Tanstack. Check out the column filtering and data editing features. </p>
+                    <p> Tanstack is <b>free to use</b> and is available on npm. Its documentation, supported frameworks, and fatures examples can be found here: <a href="https://tanstack.com/table/latest">Tanstack Table Documentation</a>. </p>
+                    <p> The Tegel team recommends reading the Tanstack Table documentation to get started and understand how to use the library. Inspiration on how to use Tegel with Tanstack can be found in the demo pages linked below. </p>
```
Comment (timrombergjakobsson) [inline packages/core/src/stories/announcements/table-libraries.stories.ts:67]: Maybe just?
```suggestion
                    <p> We recommend reading the Tanstack Table documentation to get started and understand how to use the library. Inspiration on how to use Tegel with Tanstack can be found in the demo pages linked below. </p>
```

**Comment by timrombergjakobsson** [comment]
Nice @theJohnnyMe, should we mention somewhere that the styling is limited to theme Quartz?

File: packages/core/src/stories/announcements/table-libraries.stories.ts (hunk 2 above)
Comment (theJohnnyMe) [reply packages/core/src/stories/announcements/table-libraries.stories.ts]: I agree. Thanks Tim for suggestion, made it so easy just to commit it asap ;)

File: packages/core/src/stories/announcements/table-libraries.stories.ts (hunk 4 above)
Comment (theJohnnyMe) [reply packages/core/src/stories/announcements/table-libraries.stories.ts:67]: Agree!

**Comment by theJohnnyMe** [comment]
> Great work! Should we update the links to the demo pages once the AG-Grid and TanStack examples have been merged?

Great idea! I will try to merge code in demo pages so we can already link to it.

**Comment by theJohnnyMe** [comment]
> Nice @theJohnnyMe, should we mention somewhere that the styling is limited to theme Quartz?

Good point, will do that ;) 
It also reminds me that maybe we will need to update overwrites file according to new theme builder.

**Comment by theJohnnyMe** [comment]
@nathalielindqvist and @timrombergjakobsson  - Updates done!

**Review by Lunkan89 — APPROVED** [review]

**Pull request merged**